builder/package.json

```
@@ -8,14 +8,14 @@
     "build": "tsx style-dictionary/transform.ts"
   },
   "devDependencies": {
-    "@tokens-studio/sd-transforms": "^1.3.0",
-    "@tsconfig/node20": "^20.1.5",
-    "style-dictionary": "^4.4.0",
-    "tsx": "^4.19.3",
-    "typescript": "^5.8.3"
+    "@tokens-studio/sd-transforms": "^2.0.1",
+    "@tsconfig/node20": "^20.1.6",
```

```suggestion
    "@tsconfig/node22": "^22.0.2",
```

ここも変えて良さそう〜
https://www.npmjs.com/package/@tsconfig/node22

---

お、本当だ。こちらも変更しておきます〜。